IShareApi/IShare.DAL/SqlHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace IShare.DAL
+{
+    public class SqlHelper
+    {
+        private string P_addr;
```

Comment:
`P_addr` does not conform to the c# programming convention. Check here for detail : https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/identifier-names